advanced/Scripts/database_migration/gravity/1_to_2.sql

```diff
@@ -1,4 +1,4 @@
-CREATE TABLE auditlist
+CREATE TABLE domain_auditlist
```

Comment:
This is for consistency as well. The other lists are also called `whitelist`, `blacklist`, ... and not `white`, `black`, ... although they serve a similar purpose.

Reply:
Technically, every table is a "list" of things. We used the terms whitelist and blacklist because that's what those "lists" are conventionally called, but we didn't use "list" for the regex or gravity tables. Also, the whitelist and blacklist are directly viewed by the user, whereas the audited domains are just a way for us to keep track of what domains the user has already audited. The user does not know about any "audit list", just that when they audit a domain they don't see it show up in that view again.

It's not a huge deal either way and won't block the merge, but it is confusing to use the "list" ending for two different concepts.